app/eslint.config.mjs

```diff
@@ -0,0 +1,49 @@
+import path from "node:path";
```

Comment:
New eslint config file format. Converted mostly automatically using eslint's automatic migration tool, with a few manual tweaks after. Should be equivalent to old config, though I think I added a rule or two.